src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -106,12 +106,16 @@
 {%- endblock dateinterval_widget %}
 
 {% block percent_widget -%}
+    {% if symbol == true %}
```

Comment:
let's shorten this: `{% if symbol %}` (and everywhere else below too)

Reply:
It also seems like we need to take whitespace control (`i.e.` the `-` character) into account here too.